training_scripts/gsgnn_mt/ml_nc_ec_er_lp.yaml

```diff
@@ -0,0 +1,99 @@
+---
+version: 1.0
+gsf:
+  basic:
+    backend: gloo
+    verbose: false
+    save_perf_results_path: null
+  gnn:
+    model_encoder_type: rgcn
+    fanout: "4"
+    num_layers: 1
+    hidden_size: 32
+    use_mini_batch_infer: true
+  input:
+    restore_model_path: null
+  output:
+    save_model_path: null
+    save_embed_path: null
+  hyperparam:
+    dropout: 0.
+    lr: 0.001
+    lm_tune_lr: 0.0001
+    num_epochs: 3
+    wd_l2norm: 0
+    no_validation: false
+  rgcn:
+    num_bases: -1
+    use_self_loop: true
+    sparse_optimizer_lr: 1e-2
+    use_node_embeddings: false
+  multi_task_learning:
+    - node_classification:
+      target_ntype: "movie"
+      label_field: "label"
+      multilabel: false
+      num_classes: 19
+      batch_size: 16 # will overwrite the global batch_size
```

Comment:
Why does the comment say this "will overwrite the global batch_size"?
There is no the global batch_size setting in the above configurations.

Reply:
There is a default batch_size.
If we set batch_size here, the task will use batch size 16 instead of the default one.